neuralforecast/losses/pytorch.py

```diff
@@ -635,20 +659,20 @@ def __call__(
         if mask is None:
             mask = torch.ones_like(y_hat)
 
-        error = y_hat - y.unsqueeze(-1)
+        n_q = len(self.quantiles)
 
+        error = y_hat - y.unsqueeze(-1)
         sq = torch.maximum(-error, torch.zeros_like(error))
         s1_q = torch.maximum(error, torch.zeros_like(error))
-        loss = self.quantiles * sq + (1 - self.quantiles) * s1_q
+        mqloss = self.quantiles * sq + (1 - self.quantiles) * s1_q
 
+        # Match y/weights dimensions and compute weighted average
+        mask = mask / torch.sum(mask)
         mask = mask.unsqueeze(-1)
-        wmqloss = _divide_no_nan(
-            torch.sum(loss * mask, axis=-2),
-            torch.sum(torch.abs(y.unsqueeze(-1)) * mask, axis=-2),
-        )
-        return torch.mean(wmqloss)
+        mqloss = (1 / n_q) * mqloss * mask
+        return torch.sum(mqloss)
 
-# %% ../../nbs/losses.pytorch.ipynb 60
+# %% ../../nbs/losses.pytorch.ipynb 59
 def weighted_average(
```

Comment:
Can we rescue something from this `weighted_average` function?
Or will we deprecate it?